src/Datadog.Trace.ClrProfiler.Native/metadata_builder.cpp

```diff
@@ -70,7 +70,10 @@ HRESULT MetadataBuilder::FindWrapperTypeRef(
         assembly_import_, method_replacement.wrapper_method.assembly.name);
     if (assembly_ref == mdAssemblyRefNil) {
       // TODO: emit assembly reference if not found?
-      return S_FALSE;
+      LOG_APPEND("Assembly reference for "
```

Comment:
This should never happen since we emit the assembly reference before this, but if it happens, we should log it and fail so we don't try to emit the wrong call to a global (no scope) method and crash the app.